nuxt.config.js

```diff
@@ -44,6 +46,29 @@ export default {
 			{ rel: "icon", type: "image/png", href: "/favicon.png" }
 		]
 	},
+	render: { resourceHints: false },
+	hooks: {
+		// remove uneeded js scripts
+		"generate:page": (page) => {
+			const doc = cheerio.load(page.html);
+			doc("body script[src='/_nuxt/aac6f7dcb0388507b165.js']").remove();
```

Comment:
Removing javascript on static pages using hardcoded nuxt generated names is not reliable as the script names could change from build to build. I did get a different set of names on my computer.

I wonder if you really need this `hooks` block to remove javascript from static pages? When testing this pull request, I noticed the fetch of the data comes from the local payloads that are saved in the `dist` directory with your addition of `payload` field in the nuxt.config.js `generate` block, which should be enough to deal with the remote fetching problem.

Reply:
Yes, you are absolutely right. It is enough to deal with the remote fetching problem.

I included the removal of js scripts to show what is possible and different directions we can go. I will expand on this further in the check-in today.